public/controllers/management/components/management/ruleset/ruleset-table.js

```diff
@@ -140,8 +172,8 @@ class WzRulesetTable extends Component {
       `${this.paths[this.props.request]}${showingFiles ? '/files' : ''}`,
       { params: this.buildFilter() },
     ).catch((error) => {
-      console.warn(`Error when get the items of ${section}: `, error);
-      return {};
+      error.message = `Error when get the items of ${section}`;
```

Comment:
This line broke on runtime and produce a new error, but you can create a new Error with this message like this:
```
throw new Error(`Error when get the items of ${section}`)
```
🤓